package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interlay/polkabtc",
-  "version": "0.8.0",
+  "version": "0.8.1",
```

Comment:
This should be 0.8.2, as 0.8.1 has already been merged to `dev` and will probably be released before this commit

Reply:
Good call, gonna keep it in mind to pull dev before bumping a version